src/components/net-neutrality/call-tool.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import classnames from 'classnames';
+import CallButton from './call-button.js';
+import Social from './social.js';
+
+/**
+ * This function is responsible for mapping HTTP status numbers to
+ * specific string ids for generating page content to inform the user
+ * about what that status code means in terms of their ability to call.
+ */
+function getCustomError(status) {
+  if (status === -1 || status >= 500) {
+    return ['We’re sorry, we seem to be having some trouble.', 'Please try again later.'];
+  }
+  if (status === 400) {
+    return ['We can’t call that number.'];
+  }
+  if (status === 429) {
+    return ['Thanks for your calls! You’ve reached the hourly limit. Please try again in an hour.'];
```

Comment:
Who's reached the hourly limit? Us or the user?

Reply:
dunno.

Reply:
These status codes are coming from CallPower, right?